apps/easy-messe/components/OfferMass/LetOfferMass.tsx

```diff
@@ -207,7 +209,14 @@ export default function LetOfferMass({ handleIndexTab }: LetOfferMassProps) {
                     }]);
             if (handleIndexTab) handleIndexTab(0)
         },
-        validationSchema: LetOfferMassSchema
+        validationSchema: yup.object().shape({
+            phone: yup.number(),
+            dateTime: yup.string().required('dateTimeChecked'),
+            intention: yup
+                .string()
+                .required('intensionChecked')
+                .max(300, 'intentionNumberChecked')
```

Comment:
the error messages should be `formatMessages` not keys.
what's preventing you from using the formatMessage here directly?